### PR TITLE
add pivot news story

### LINKED ISSUE #576: add "news" item for "tiktok pivot"

### PR DESCRIPTION
**Pull request by akhrail1996**
closes #576

### CODE REVIEW

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:172]: should be dynamic

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:36]: a bit abrupt

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:26]: did old Since have a different mission? It's not clear that it did.

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:57]: why capitalized Story?

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:21]: `alignment`, `corner_radius`, `zoom`, when `0`, should be omitted.

File: priv/gettext/en/LC_MESSAGES/news.po
Comment (ruslandoga) [inline priv/gettext/en/LC_MESSAGES/news.po:57]: do we actually have any english speakers btw? Either way, I'd say something like "only your matches would see".

File: priv/gettext/en/LC_MESSAGES/news.po
Comment (ruslandoga) [inline priv/gettext/en/LC_MESSAGES/news.po:79]: fuzzy

File: test/t_web/channels/feed_channel_test.exs
Comment (ruslandoga) [inline test/t_web/channels/feed_channel_test.exs:448]: this is not testing much, you need to at least keep the response structure test

**Review by ruslandoga — CHANGES_REQUESTED** [review]

File: lib/t/news.ex
Comment (akhrail1996) [reply lib/t/news.ex:172]: should it use sth like this: https://github.com/getsince/test3/blob/master/lib/t_web/views/view_helpers.ex#L60?

File: lib/t/news.ex
Comment (akhrail1996) [reply lib/t/news.ex:36]: Abrupt meaning sudden or rude?
image

File: lib/t/news.ex
Comment (akhrail1996) [reply lib/t/news.ex:26]: yeah, the key difference in this pivot in audio -> contacts, should articulate it better

File: lib/t/news.ex
Comment (akhrail1996) [reply lib/t/news.ex:57]: kinda putting greater emphasis on the Story, we did in some other places as well

File: lib/t/news.ex
Comment (ruslandoga) [reply lib/t/news.ex]: yes

File: lib/t/news.ex
Comment (ruslandoga) [reply lib/t/news.ex]: sudden

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:22]: что поменялось?

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:27]: выше нужно написать, что поменялось, тут нужно указать, что будет дальше. Можно использовать больше одного слова.

File: lib/t/news.ex
Comment (ruslandoga) [reply lib/t/news.ex:42]: it kinda sucks

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:46]: нажми, и? нужно указать что это, "Вот наши"

File: lib/t/news.ex
Comment (ruslandoga) [inline lib/t/news.ex:122]: как? если нужно свайпать, об этом нужно написать

**Review by ruslandoga — CHANGES_REQUESTED** [review]
Sorry, I can't approve it. You can merge it without my approval.

File: lib/t_web/channels/feed_channel.ex
Comment (ruslandoga) [inline lib/t_web/channels/feed_channel.ex:57]: no

File: lib/t_web/channels/feed_channel.ex
Comment (ruslandoga) [inline lib/t_web/channels/feed_channel.ex:546]: no

File: lib/t_web/views/feed_view.ex
Comment (ruslandoga) [inline lib/t_web/views/feed_view.ex:27]: no

**Pull request merged**
